src/controllers/payment.callback.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response } from "express";
+import { urlUtils } from "../utils/url";
+import { logger } from "../utils/logger";
+import { CONFIRMATION_PATH, REVIEW_PATH } from "../types/page.urls";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const paymentStatus = req.query.status;
+    if (paymentStatus === "paid") {
+      logger.info("Payment status: " + paymentStatus + " - redirecting to the confirmation page");
```

Comment:
Could we either add some sort of identifier to these logs (submission id maybe?) or set them to debug